pkg/grpc/device_definition.proto

```diff
@@ -268,6 +268,7 @@ message Feature {
 message DeviceCompatibilities {
 	int32 year = 1;
 	repeated Feature features = 2;
+	string level = 3;
```

Comment:
thought: Any value to using [`enum`](https://developers.google.com/protocol-buffers/docs/proto3#enum) here?

It's attractive because the caller wants one of three values, but maybe it is too fussy.

Reply:
could be our first enum use case in gRPC protos!